computer-configration-website/src/app/lib/utils.js

```
@@ -0,0 +1,19 @@
+/**
+ *
+ * @param {*} obj
+ * @returns Json object with the id key removed
+ */
+function removeIdToJson(obj) {
+  let newObj = {};
+  delete Object.assign(newObj, obj).id;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)